lang/src/org/partiql/lang/errors/ErrorCode.kt

```diff
@@ -531,6 +531,12 @@ enum class ErrorCode(
         "Other expressions may not be present in the select list when '*' is used without dot notation."
     ),
 
+    PARSE_INVALID_ARGUMENTS_FOR_TRIM(
```

Comment:
Probably change the name to `PARSE_INVALID_TRIM_SPEC`? The term "arguments" can refer to the string being trimmed.

Reply:
Addressed in the new commit.